app/models/form/sales/subsections/household_characteristics.rb

```diff
@@ -14,10 +14,10 @@ def pages
       (Form::Sales::Pages::PrivacyNotice.new("privacy_notice", nil, self, joint_purchase: false) unless form.start_year_after_2024?),
       Form::Sales::Pages::Age1.new(nil, nil, self),
       Form::Sales::Pages::RetirementValueCheck.new("age_1_retirement_value_check", nil, self, person_index: 1),
+      (Form::Sales::Pages::NotRetiredValueCheck.new("age_1_not_retired_value_check", nil, self, person_index: 1) if form.start_year_after_2024?),
       Form::Sales::Pages::OldPersonsSharedOwnershipValueCheck.new("age_1_old_persons_shared_ownership_joint_purchase_value_check", nil, self, joint_purchase: true),
       Form::Sales::Pages::OldPersonsSharedOwnershipValueCheck.new("age_1_old_persons_shared_ownership_value_check", nil, self, joint_purchase: false),
       Form::Sales::Pages::GenderIdentity1.new(nil, nil, self),
-      Form::Sales::Pages::RetirementValueCheck.new("gender_1_retirement_value_check", nil, self, person_index: 1),
```

Comment:
Retirement age, and thus this validation, no longer depends on gender (and that's set up to be true regardless of form year) so there's no point in this page being here.